File: Algorithm/보통의_취준생을_위한_코딩테스트_with_파이썬/06.List/6-1-1_boj_10818.py
'''----------------------------------------------------
Sub  : [BOJ] 최소, 최대
Link : https://www.acmicpc.net/problem/10818
Level: 브론즈 3
Tag  : Python, 
Memo
----------------------------------------------------'''

import sys
input = sys.stdin.readline

# 입력
n = int(input().rstrip())
arr_list = list(map(int, input().rstrip().split()))

max_num = arr_list[0]
min_num = arr_list[0]

for num in arr_list:
    max_num = num if num > max_num else max_num
    min_num = num if num < min_num else min_num

print(min_num, max_num)

File: Algorithm/보통의_취준생을_위한_코딩테스트_with_파이썬/06.List/6-1-1_boj_10818.c
/*-----------------------------------------------------
Sub  : [BOJ] 최소, 최대
Link : https://www.acmicpc.net/problem/10818
Level: 브론즈 3
Tag  : C,
Memo
-----------------------------------------------------*/

#include <stdio.h>

#define MAX 1000000

int main() {
    int n;
    int arr[MAX];
    int num_max = -MAX;
    int num_min = MAX;

    // 입력
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
        if (arr[i] > num_max) num_max = arr[i];
        if (arr[i] < num_min) num_min = arr[i];
    }

    // 출력
    printf("%d %d\n", num_min, num_max);

    return 0;
}
